ldb_reader.go

```diff
@@ -50,6 +50,23 @@ func (reader *LDBReader) GetLastSequence(ctx context.Context) (schema.DMLSequenc
 	return ldb.FetchSeqFromLdb(ctx, reader.Db)
 }
 
+type Latencier interface {
+	GetLedgerLatency(ctx context.Context) (time.Duration, error)
+}
+
+type MockLatencier struct {
```

Comment:
I'd rather not make this part of the public API. Would be curious where I might want to consider moving it to (a `pkg/latencier` package?).